src/repositories/ads.repository.ts

```diff
@@ -0,0 +1,18 @@
+import { Repository } from '../core/repository';
+
+export class AdsRepository extends Repository {
+  public async getPostInsights(id: string) {
```

Comment:
The name of this method must be `graphql` 😄

Reply:
Hmm, but graphql is endpoint for a lot different requests. 

Should I then rename it to graphql and implament in services or entities options for doc_id, etc?

Reply:
I'm not familiar with this endpoint parameters, but it seems like you should. At least it should be placed in `ads.graphql` 100%.
Just watch how it made for direct messaging. Hope it will be clear

https://github.com/dilame/instagram-private-api/blob/master/src/entities/direct-thread.entity.ts

https://github.com/dilame/instagram-private-api/blob/master/src/repositories/direct-thread.repository.ts

Reply:
BTW, I will be grateful if you could explain this endpoint possibilities.